generated/nidcpower/nidcpower_service.cpp

```diff
@@ -3354,7 +3466,8 @@ namespace nidcpower_grpc {
     try {
       auto vi_grpc_session = request->vi();
       ViSession vi = session_repository_->access_session(vi_grpc_session.name());
-      auto channel_name = request->channel_name().c_str();
+      auto channel_name_mbcs = convert_from_grpc<std::string>(request->channel_name());
```

Comment:
You should also update MeasureMultiple and MeasureMultipleLCR: https://github.com/ni/grpc-device/blob/main/source/custom/nidcpower_service.custom.cpp

In general, you should update the relevant custom entry points: https://github.com/ni/grpc-device/tree/main/source/custom

Reply:
Done